From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Groldi <chniucg@gmail.com>
Date: Fri, 11 Aug 2023 20:32:00 +0200
Subject: [PATCH] Update build.gradle.kts dependency path

The dependency path for the "kissenpaper-api" project was corrected in the "KissenPaper-Server" build.gradle.kts file. Previously, it was pointing to an incorrect location, causing build errors. Now it points to the correct sub-project within "kissenpaper", fixing the issue.

diff --git a/build.gradle.kts b/build.gradle.kts
index 71a806fe93614cf2160b41f30a06b72f3cc0dab2..4edda159b12943f83994a361a6236292487295b2 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -14,7 +14,7 @@ val alsoShade: Configuration by configurations.creating
 
 dependencies {
     // KissenPaper start
-    implementation(project(":kissenpaper-api"))
+    implementation(project(":kissenpaper:kissenpaper-api"))
     implementation("io.papermc.paper:paper-mojangapi:${project.version}") {
         exclude("io.papermc.paper", "paper-api")
     }
